src/postgres/commands/createPostgresServer/IPostgresServerWizardContext.ts

```diff
@@ -3,24 +3,24 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { Sku } from "@azure/arm-postgresql/esm/models";
 import { Server } from "@azure/arm-postgresql/src/models";
 import { IAzureDBWizardContext } from "../../../tree/IAzureDBWizardContext";
 
 export interface IPostgresServerWizardContext extends IAzureDBWizardContext {
+  /**
+   * Username without server, i.e. "user1"
+   */
+  shortUserName?: string;
+  /**
+   * Username with server, i.e. "user1@server1"
+   */
+  longUserName?: string;
+  adminPassword?: string;
 
-    /**
-     * Username without server, i.e. "user1"
-     */
-    shortUserName?: string;
-    /**
-     * Username with server, i.e. "user1@server1"
-     */
-    longUserName?: string;
-    adminPassword?: string;
-
-    addFirewall?: boolean;
-    publicIp?: string;
-
-    server?: Server;
+  addFirewall?: boolean;
+  publicIp?: string;
```

Comment:
It looks like the tabbing for these lines changed to width 2. Can you revert it back to width 4?